src/ReaderScreen/utils/index.js

```diff
@@ -112,7 +112,8 @@ export const loadHTML = (
         const sequencesData = item.sequences
           ? ` data-sequences='|${item.sequences.join("|")}|'`
           : "";
-        let contentHtml = `<div id="${paragraphId}" class='text-item'${sequencesData}>`;
+        const sequenceData = ` data-sequence='${paragraphId}'`;
+        let contentHtml = `<div id="${item.id}" class='text-item'${sequenceData}${sequencesData}>`;
```

Comment:
The single quote ends after `text-item` classname, do we want to include the sequenceData to the classname?

Shouldn't it be
```js
let contentHtml = `<div id="${item.id}" class='text-item${sequenceData}${sequencesData}'>`;
```

Reply:
no we are not adding sequence data to class name its new attribute.